bonsai/Cargo.toml

```diff
@@ -1,14 +1,8 @@
 [workspace]
 resolver = "2"
-members = [
-  "ethereum",
-  "ethereum-relay",
-  "rest-api-mock",
-  "sdk",
-]
+members = ["ethereum", "ethereum-relay", "rest-api-mock", "sdk"]
 
 [workspace.package]
-version = "0.17.0"
 edition = "2021"
 license = "Apache-2.0"
 homepage = "https://risczero.com/"
```

Comment:
Not release-blocking, but should some of the rest of this information be made Bonsai-specific (e.g. should we point to the Bonsai website here?)